grafana-plugin/src/pages/incident/Incident.tsx

```diff
@@ -395,7 +395,7 @@ class IncidentPage extends React.Component<IncidentPageProps, IncidentPageState>
             </li>
           ))}
         </ul>
-        <Field label="Leave a resolution note" description="Will also show up in the thread of incident in Slack">
+        <Field label="Leave a resolution note" description="Will also show up in the thread of alert group in Slack">
```

Comment:
Shouldn't this _be show up in the thread of **the** alert group_ or _should up in the thread of **an** alert group_ ? It is not correct as it is right now